SimpleCalculator

```diff
@@ -0,0 +1,134 @@
+//HTML CODE FOR THE CALCULATRO
```

Comment:
`<!--HTML CODE FOR THE CALCULATOR-->` 
I think you should use this for comment.